org.contour.Terminal.yml

```
@@ -0,0 +1,92 @@
+app-id: org.contour.Terminal
+name: Contour Terminal Emulator
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: contour
+finish-args:
+  - --filesystem=host
+  - --device=all          # we need access to at least DRI (for OpenGL) & PTY
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

assuming proper wayland support

---

I prefer it to remain fallback-x11 for the time being, because I know that works more well. In fact, it's an OpenGL based app that is fully utilizing Qt APIs, and yet I sometimes have struggles and I even files one bug report to official Qt bug tracker due to some infinite recursion that I detected. 

So with this in mind, I think it should favor X11 until I am more confident.

---

the thing it is currently NOT `fallback-x11`

---

Okay, I've removed the wayland-socket then, and I assume that it'll then take the X11 protocol when on Wayland desktop, right?

---

absolutely not. This is not what I was suggesting.

---

that's what it's supposed to do. Unfortunate but it's Qt.